<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Iniciar Treinamento</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbCaminhoArquivo">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Caminho arquivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>656</x>
      <y>2</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVideoProcessado">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>420</width>
      <height>320</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>420</width>
      <height>320</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>420</width>
      <height>320</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Video Processado</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVideoOriginal">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>210</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>210</width>
      <height>160</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>210</width>
      <height>160</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(0, 0, 0)</string>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>401</width>
      <height>23</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="tbTrainningPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>341</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Caminho arquivo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btTrainningPath">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>332</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btClearTrainning">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar Treinamento</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStatusMsg">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamanho DB:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDatabaseSize">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btSearchDatabase">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Pesquisar</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>256</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1208</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
